docker/batch_user.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
Maybe change one of the script names? Two shell scripts with different content can get confusing.